launch/test/test_launch_args.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from launch.arguments import get_launch_args
+
+
+def test_launch_args():
+    # Test empty arguments
+    argv = []
+    args = get_launch_args(argv)
+    assert len(args) == 0, "Failed empty list"
```

Comment:
Please use single quote throughout the file where possible (when no `'` are in the string which would require quoting).